### ISSUE TITLE
Failed to register simple pkh(@0/**) policy

### ISSUE THREAD

**Issue by edouardparis** [LedgerHQ/app-bitcoin-new#129]
From https://github.com/wizardsardine/async-hwi/issues/29

This policy does not require registration, but still the registration failed. 

Using speculos:
```
SPECULOS_APPNAME="Bitcoin Test:2.1.0" ./speculos.py apps/nanos-2.1-bitcoin-test-2.1.0-rc.elf --model nanos
```
and https://github.com/bigspider/HWI/
```
bigspider-hwi on  wallet-policies [?] is 📦 v2.1.1 via 🐍 v3.9.2 (venv) 
✦ ❯ python3 -m hwi --chain test --device-type ledger getxpub "m/44'/1'/0'"
{"xpub": "tpubDCwYjpDhUdPGP5rS3wgNg13mTrrjBuG8V9VpWbyptX6TRPbNoZVXsoVUSkCjmQ8jJycjuDKBb9eataSymXakTTaGifxR6kmVsfFehH1ZgJT"}

bigspider-hwi on  wallet-policies [?] is 📦 v2.1.1 via 🐍 v3.9.2 (venv) 
✦ ❯ python3 -m hwi --chain test --device-type ledger displayaddress\
       --policy "pkh(@0/**)"\
       --name "Ledger"\
       --keys "[\"[f5acc2fd/44'/1'/0']tpubDCwYjpDhUdPGP5rS3wgNg13mTrrjBuG8V9VpWbyptX6TRPbNoZVXsoVUSkCjmQ8jJycjuDKBb9eataSymXakTTaGifxR6kmVsfFehH1ZgJT\"]"
{"address": "mz5vLWdM1wHVGSmXUkhKVvZbJ2g4epMXSm"}

bigspider-hwi on  wallet-policies [?] is 📦 v2.1.1 via 🐍 v3.9.2 (venv) 
✦ ❯ python3 -m hwi --chain test --device-type ledger registerpolicy\      
       --policy "pkh(@0/**)"\
       --name "Ledger"\
       --keys "[\"[f5acc2fd/44'/1'/0']tpubDCwYjpDhUdPGP5rS3wgNg13mTrrjBuG8V9VpWbyptX6TRPbNoZVXsoVUSkCjmQ8jJycjuDKBb9eataSymXakTTaGifxR6kmVsfFehH1ZgJT\"]" 
{"error": "('0x6a82', 'Error in <BitcoinInsType.REGISTER_WALLET: 2> command', '')", "code": -13}
```

**Comment by bigspider** [comment]
I tried to register the exact same policy as a test in `test_register_wallet.py`:

```python
@has_automation("automations/register_wallet_accept.json")
def test_register_wallet_standard_policy(client: Client, speculos_globals):
    run_register_test(client, speculos_globals, WalletPolicy(
        name="Ledger",
        descriptor_template="pkh(@0/**)",
        keys_info=[
            "[f5acc2fd/44'/1'/0']tpubDCwYjpDhUdPGP5rS3wgNg13mTrrjBuG8V9VpWbyptX6TRPbNoZVXsoVUSkCjmQ8jJycjuDKBb9eataSymXakTTaGifxR6kmVsfFehH1ZgJT",
        ],
    ))
```

and it worked fine. Tried both on the current `develop` head, and on the [2.1.0 tag](https://github.com/LedgerHQ/app-bitcoin-new/releases/tag/2.1.0), running the test on speculos on model Nano S.

**Comment by bigspider** [comment]
Thanks for confirming; leaving it open in case @wszdexdrf has something to add, but looks like this was fixed by #80, and therefore it should already be solved in production since version 2.1.0 of the app.

**Comment by edouardparis** [comment]
Case close https://github.com/wizardsardine/async-hwi/issues/29
Thank you very much for your help
